nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -84,6 +84,8 @@ while [ "$#" -gt 0 ]; do
     esac
 done
 
+extraBuildFlags+=('--option' 'min-free' '0')
```

Comment:
We probably don't need this either, but it's harmless to keep it for now.  We ended up having to disable the AMI build that required this workaround anyway because AMIs are expensive to build in CI.